### PR TITLE
Home styles and logic

### PR DESCRIPTION
**Pull request by Nick2603**

### CODE REVIEW

File: src/_variables.tsx [hunk 1]
```diff
@@ -0,0 +1,7 @@
+export const variables = {
```
Comment (Hopchenko) [inline src/_variables.tsx:1]: do we need this?

Comment (Hopchenko) [inline src/_variables.tsx:1]: we dont need underscore for js files :) 
https://stackoverflow.com/questions/34889962/why-put-in-front-of-the-file-name-or-in-scss-css

File: src/_variables.scss [hunk 2]
```diff
@@ -2,4 +2,4 @@ $primary-color: #ff5a5f;
 $secondary-color: #79cccd;
```
Comment (Hopchenko) [inline src/_variables.scss:2]: Can you create a styles folder and put this file in it?

**Review by Hopchenko — CHANGES_REQUESTED** [review]

File: src/components/SocialMedia/SocialMedia.tsx [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+export const SocialMedia: FC = () => {
+  const linkDetails: ILinkDetails[] = [
+    { href: "https://www.facebook.com/", src: fbIcon, alt: "facebook icon" },
+    {
+      href: "https://www.instagram.com/",
+      src: instagramIcon,
+      alt: "instagram icon",
+    },
+    { href: "https://twitter.com/", src: twitterIcon, alt: "twitter icon" },
+    { href: "http://vk.com/", src: vkIcon, alt: "vk icon" },
```
[the remark this answers is not in the document]
Comment (Hopchenko) [reply src/components/SocialMedia/SocialMedia.tsx:24]: image

File: src/components/CountrySelect/CountrySelect.tsx [hunk 4]
```diff
@@ -1,31 +1,44 @@
 import React, { FC, useState } from "react";
+import { useNavigate } from "react-router-dom";
 import TextField from "@mui/material/TextField";
 import Autocomplete from "@mui/material/Autocomplete";
 import { CountryList } from "./CountryList";
+import { MyButton } from "../MyButton/MyButton";
 
 export const CountrySelect: FC = () => {
   const [value, setValue] = useState<string | null>(CountryList[0]);
```
[the remark this answers is not in the document]
Comment (Nick2603) [reply src/components/CountrySelect/CountrySelect.tsx:9]: This is just my custom button. I make a component of it so I do not have to paste MU code each time. Can I call it MyCustomeButton?

File: src/components/CountrySelect/CountrySelect.tsx [hunk 5]
```diff
@@ -1,31 +1,44 @@
...
+          }}
+          id="controllable-states-demo"
+          options={CountryList}
+          sx={{ width: 300 }}
+          renderInput={(params) => (
+            <TextField {...params} label="Choose your country" />
+          )}
+        />
+      </div>
+      <MyButton navigate={navigateToCountryPage} />
```
[the remark this answers is not in the document]
Comment (Nick2603) [reply src/components/CountrySelect/CountrySelect.tsx:41]: This is my custom button which I am going to use all across the project.

[the remark this answers is not in the document]
Comment (Nick2603) [reply src/components/CountrySelect/CountrySelect.tsx:41]: This button is done based on MU example and I decided to make a component as I am reusing it across the project.

File: src/components/Navigation/Navigation.module.scss [hunk 6]
```diff
@@ -12,7 +12,13 @@
...
+
   a:hover {
-    color: $primary-color;
+    font-weight: bold;
+    text-shadow: 2px 2px 2px #ce5937;
   }
 }
+
+.active-link {
+  font-weight: bold !important;
```
[the remark this answers is not in the document]
Comment (Nick2603) [reply src/components/Navigation/Navigation.module.scss:23]: This style doesn't work without important

File: src/_variables.tsx (hunk 1 above)
Comment (Nick2603) [reply src/_variables.tsx:1]: I have two components where I am using it as I cannot use variables.scss inside tsx files.
I use it in MyButton.tsx and in HomePage.tsx

**Review by Hopchenko — APPROVED** [review]

**Pull request merged**